nix/install.go

```diff
@@ -43,3 +48,20 @@ func Install() error {
 
 	return errors.WithStack(cmd.Wait())
 }
+
+func EnsureInstalled(cmd *cobra.Command, args []string) error {
+	_, err := exec.LookPath("nix-shell")
+	if err == nil {
+		return nil
+	}
+
+	if featureflag.NixInstaller.Enabled() {
+		color.Yellow("\nNix is not installed. Devbox will attempt to install it. \n\nPress enter to continue.\n")
```

Comment:
Can we also add something like "... or ctrl-c to exit." so the user knows how to bail out?